src/Schema/Blueprint.php

```diff
@@ -295,6 +315,54 @@ public function dropRowDeletionPolicy(): Fluent
         return $this->addCommand('dropRowDeletionPolicy');
     }
 
+    /**
+     * @param string $name
+     * @return SequenceDefinition
+     */
+    public function createSequence(string $name): SequenceDefinition
+    {
+        $this->commands[] = $command = new SequenceDefinition(__FUNCTION__, $name);
+        return $command;
+    }
+
+    /**
+     * @param string $name
+     * @return SequenceDefinition
+     */
+    public function createSequenceIfNotExists(string $name): SequenceDefinition
+    {
+        $this->commands[] = $command = new SequenceDefinition(__FUNCTION__, $name);
+        return $command;
+    }
+
+    /**
+     * @param string $name
+     * @return SequenceDefinition
+     */
+    public function alterSequence(string $name): SequenceDefinition
+    {
+        $this->commands[] = $command = new SequenceDefinition(__FUNCTION__, $name);
+        return $command;
+    }
+
+    /**
+     * @param string $name
+     * @return Fluent<string, mixed>
+     */
+    public function dropSequence(string $name): Fluent
+    {
+        return $this->addCommand(__FUNCTION__, ['sequence' => $name, 'ifExists' => false]);
```

Comment:
ifExists seems to be not used in compileDropSequence / compileDropSequenceIfExists ?

Reply:
Thanks. fixed.